### ISSUE TITLE
Exception is thrown when using BigDecimal in response

### ISSUE THREAD

**Issue by victor-lyan** [bkbnio/kompendium#75]
I have a route like this:

```kotlin
route("organization") {
    notarizedGet(
        MethodInfo.GetInfo<CityIdQuery, List<OrganizationDto>>(
            summary = "Get organizations",
            tags = setOf("dicts"),
            responseInfo = ResponseInfo(
                status = HttpStatusCode.OK,
                description = "",
            )
        )
    ) {
        ...
    }
}
```

and OrganizationDto class is like this:

```kotlin
data class OrganizationDto(
  val id: UUID,
  val commission: BigDecimal? = null,
  ...
)
```

Whenever I run the project I get an exception:

```
Caused by: java.lang.IllegalStateException: Unable to parse field type from val kotlin.IntArray.size: kotlin.Int
	at io.bkbn.kompendium.Kontent.handleComplexType(Kontent.kt:141)
	at io.bkbn.kompendium.Kontent.access$handleComplexType(Kontent.kt:30)
	at io.bkbn.kompendium.Kontent$generateKTypeKontent$2.invoke(Kontent.kt:113)
	at io.bkbn.kompendium.Kontent$generateKTypeKontent$2.invoke(Kontent.kt:98)
	at io.bkbn.kompendium.util.Helpers.logged(Helpers.kt:26)
```

This is because of this line:

```kotlin
object Kontent {
  ...
  val field = prop.javaField?.type?.kotlin ?: error("Unable to parse field type from $prop")
  ...
}
```

**Comment by rgbrizzlehizzle** [comment]
Ahhh yes this is a problem, due to my kinda hacky reflection implementation… the super quick fix would to be to add class marchers for all the “big number” classes.  But perhaps there is a smarter way?

**Comment by victor-lyan** [comment]
yep, I agree. I'll create a PR for this change

**Comment by rgbrizzlehizzle** [comment]
resolved by #76
